awx/ui/src/screens/Inventory/ConstructedInventory.js

```diff
@@ -21,9 +21,9 @@ import RelatedTemplateList from 'components/RelatedTemplateList';
 import { ResourceAccessList } from 'components/ResourceAccessList';
 import RoutedTabs from 'components/RoutedTabs';
 import ConstructedInventoryDetail from './ConstructedInventoryDetail';
-import ConstructedInventoryEdit from './ConstructedInventoryEdit';
+import ConstructedInventoryEdit from './InventoryEdit';
```

Comment:
Can we change this back to `ConstructedInventoryEdit`? I don't think we'll use the regular inventory edit form for this.